include/RAJA/cuda.hxx

```diff
@@ -561,12 +561,12 @@ __device__ inline double _atomicAdd(double *address, double value)
 //
 // Headers containing traversal and reduction templates
 //
-#include "RAJA/exec-cuda/forall_cuda.hxx"
-#include "RAJA/exec-cuda/reduce_cuda.hxx"
-#include "RAJA/exec-cuda/scan_cuda.hxx"
+#include "RAJA/internal/exec-cuda/forall_cuda.hxx"
+#include "RAJA/internal/exec-cuda/reduce_cuda.hxx"
+#include "RAJA/internal/exec-cuda/scan_cuda.hxx"
 
 #if defined(RAJA_ENABLE_NESTED)
-#include "RAJA/exec-cuda/forallN_cuda.hxx"
+#include "RAJA/internal/exec-cuda/forallN_cuda.hxx"
```

Comment:
It would be nice to be able to include cuda without forallN in places where forallN is not used.